documentation/src/main/asciidoc/topics/con_protostream.adoc

```diff
@@ -0,0 +1,15 @@
+[id='protostream_{context}']
+= ProtoStream marshalling
+{brandname} includes a ProtoStream API that marshalls Java objects as Protocol Buffers (Protobuf).
+
+ProtoStream natively supports many different Java data types, which means you do not need to configure ProtoStream marshalling for those types.
+For custom or user types, you need to provide some information so that {brandname} can marshall those objects to and from your caches.
+
+`SerializationContext`:: A repository that contains Protobuf type definitions, loaded from Protobuf schemas (`.proto` files), and the accompanying marshallers.
+
+`SerializationContextInitializer`:: An interface that initializes a `SerializationContext`.
+
+[role="_additional-resources"]
+.Additional resources
+* link:{protostreamdocroot}/org/infinispan/protostream/SerializationContext.html[org.infinispan.protostream.SerializationContext]
+* link:{protostreamdocroot}/org/infinispan/protostream/SerializationContextInitializer.html[org.infinispan.protostream.SerializationContextInitializer]
```

Comment:
both links are broken. they are being resolved as `https://www.javadoc.io/doc/org.infinispan.protostream/protostream/latest/index.html/org/infinispan/protostream/SerializationContext.html` 
if you noticed, `index.html` is included in `protostreamdocroot` which makes the browser ignoring everything else after it.